src/response.ts

```diff
@@ -64,21 +59,36 @@ document.addEventListener('DOMContentLoaded', async () => {
     (el as HTMLElement).innerText = useremail;
   }
 
+  window.addEventListener(
```

Comment:
Let's remember to open new PR on `io-pay-portal` to modify html response of `postMessage` to set `targetOrigin` to `io-pay` target